### PR TITLE
Setup linter for import aliases

### PR DESCRIPTION
**Pull request by eqrx**
### Check List Before Merging

- [ ] This PR passes all pre-commit hook validations.
- [ ] This PR is fully tested and regression tests are included.
- [ ] Relevant documentation has been updated.

### CODE REVIEW

**Comment by erdii** [comment]
This PR contains a commit that is contained in #588 right? Won't this mess with git history in weird ways?

**Comment by eqrx** [comment]
@erdii why would it? thats what git is for or not? :D

File: cmd/remote-phase-manager/main.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"k8s.io/client-go/rest"
+	"k8s.io/client-go/scale/scheme"
```
Comment (erdii) [inline cmd/remote-phase-manager/main.go:17]: This imports another package now. Is this change on purpose or an autoimporter fail?

File: internal/packages/packagecontent/topackage_test.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/stretchr/testify/require"
 	"k8s.io/apimachinery/pkg/runtime"
 
-	pkoapis "package-operator.run/apis"
+	apis "package-operator.run/apis"
```
Comment (erdii) [inline internal/packages/packagecontent/topackage_test.go:11]:
```suggestion
	"package-operator.run/apis"
```

**Review by erdii — CHANGES_REQUESTED** [review]
Finally viewed all files :muscle: 

Fix the nit and you'll have my seal of approval :)

**Review by erdii — APPROVED** [review]

**Pull request merged**